packages/react/time-series/src/TimeSeries.tsx

```diff
@@ -257,10 +258,10 @@ export function TimeSeries(props: TimeSeriesProps) {
         }
       )
 
-      const metricData = response.metricByName.timeSeries
+      const metricData = response.timeSeries
 
-      const labels: string[] = [...metricData.labels]
-      const values: number[] = [...metricData.values]
+      const labels = metricData?.labels ?? []
+      const values = (metricData?.values ?? []).map((value) => (value === null ? null : Number(value)))
```

Comment:
```suggestion
      const values = (metricData?.values ?? []).map((value) => (value == null ? null : Number(value)))
```
```ts
const value = undefined
value === null // false
value == null // true
```

Reply:
I think we can just do:

```ts
value ? Number(value) : null
```

Since `value` type is `string | null | undefined` and it should never be an empty string.

Reply:
@GabrielMusatMestre I am making the change to be robust, although I believe `metricData?.values` should truly contain an array of `string | null`, rather than `string | null | undefined`.